tests/console/http_srv.pm

```diff
@@ -11,42 +11,37 @@
 # Summary: Simple apache server test
 # Maintainer: Oliver Kurz <okurz@suse.de>
 
+package http_srv;
+use base "services::apache";
 use strict;
 use warnings;
 use base "consoletest";
 use testapi;
 use utils;
 
+sub run_test {
+    my ($self) = @_;
+
+    $self->install_apache();
+    $self->enable_apache();
+    $self->start_apache_service();
+    $self->check_apache_service();
+    $self->check_apache_function();
+}
+
 sub run {
+    my ($self) = @_;
     select_console 'root-console';
-    # Log space available before installation, see poo#19834
     script_run("df -h > /dev/$serialdev", 0);
-    # Install apache2
-    zypper_call("in apache2");
-    # After installation, apache2 is disabled
-    systemctl 'show -p UnitFileState apache2.service|grep UnitFileState=disabled';
-
-    # let's try to run it
-    systemctl 'start apache2.service';
-    systemctl 'show -p ActiveState apache2.service|grep ActiveState=active';
-    systemctl 'show -p SubState apache2.service|grep SubState=running';
-
-    # verify httpd serves index.html
-    type_string "echo Lorem ipsum dolor sit amet > /srv/www/htdocs/index.html\n";
-    assert_script_run(
-        "curl -f http://localhost/ | grep 'Lorem ipsum dolor sit amet'",
-        timeout      => 90,
-        fail_message => 'Could not access local apache2 instance'
-    );
+    $self->run_test();
```

Comment:
I don't see the benefit of having a method `run` that only calls `run_test`, why not move the content from `run_test` in here instead?